src/Cluster/RedisStrategy.php

```diff
@@ -55,4 +55,15 @@ public function getDistributor()
             'This cluster strategy does not provide an external distributor'
         );
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getDefaultCommands()
+    {
+        $commands = parent::getDefaultCommands();
+        $commands['WAIT'] = array($this, 'getKeyFromFirstArgument');
```

Comment:
I'm not convinced that this is the right approach to solve #298, the first argument of `WAIT` is not a key so it doesn't make much sense. I'm going to elaborate on this in a comment to your PR.